scriptworker/test/test_integration.py

```diff
@@ -164,7 +165,7 @@ def get_temp_creds(context):
         assert result['status']['state'] == 'pending'
         async with remember_cwd():
             os.chdir(os.path.dirname(context.config['work_dir']))
-            status = await worker.run_tasks(context, creds_key="integration_credentials")
+            status = await worker.run_tasks(context)
```

Comment:
I removed these due to the parameter [not being used in the `run_tasks(...)` function](https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/worker.py#L94-L132)

Reply:
You're right. Looks like it's been useless since https://github.com/mozilla-releng/scriptworker/commit/9b8dcb3f5f8e8f3ff5f76927694de972ad769334#diff-9baccc65625cbceb2579ef438f4e18c2 =\

Reply:
Based on the top of `test_integration.py`, it's definitely [using `integration_credentials` from the `secrets.json` file](https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/test/test_integration.py#L40) :)